x-pack/docs/en/security/authentication/realm-chains.asciidoc

```diff
@@ -23,11 +23,13 @@ after several successive failed login attempts. If the same username exists in
 multiple realms, unintentional account lockouts are possible. For more
 information, see <<trouble-shoot-active-directory>>.
 
-The default realm chain contains the `native` and `file` realms. To explicitly
+The default realm chain contains the `file` and `native` realms. To explicitly
 configure a realm chain, you specify the chain in the `elasticsearch.yml` file.
-When you configure a realm chain, only the realms you specify are used for
-authentication. To use the `native` and `file` realms, you must include them in
-the chain.
+If your realm chain does not contain `file` or `native` realm or does not disable
+them explicitly, `file` and `native` realms will be added automatically to the
+beginning of the realm chain in that order. To opt-out from the automatic behaviour,
+you can explicitly configure the `file` and `native` realms with the `order`
+and `enabled` settings.
```

Comment:
I'd like us to include an example here (or somewhere else in the docs), like:

> This sample enables the file realm and an LDAP realm, but disables the native realm:
> ```
> xpack.security.authc.realms:
>   file.file_realm:
>     order: 0
>
>   ldap.corporate_ldap
>     order: 1
>     url: 'url_to_ldap_directory'
>     ...
>
>   native.native_realm:
>     enabled: false
> ```

Reply:
I updated the existing sample to disable the native realm as an example. Thanks!